modules/era/sql/era_droplist.sql

```diff
@@ -17774,20 +17790,23 @@ INSERT INTO `mob_droplist` VALUES (2202,0,0,1000,19213,@UNCOMMON); -- White Tath
 INSERT INTO `mob_droplist` VALUES (2202,0,0,1000,901,20);          -- Venomous Claw (2.0%)
 
 -- ZoneID: 200 - Serket
-INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,4172,@ALWAYS);  -- Reraiser (Always, 100%)
-INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,4173,@VCOMMON); -- Hi-Reraiser (Very Common, 24%)
-INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,4174,@VCOMMON); -- Vile Elixir (Very Common, 24%)
-INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,901,@VCOMMON);  -- Venomous Claw (Very Common, 24%)
-INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,4175,@COMMON);  -- Vile Elixir +1 (Common, 15%)
-INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,12348,@COMMON); -- Serket Shield (Common, 15%)
-INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,16767,@COMMON); -- Triple Dagger (Common, 15%)
-INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,13552,@RARE);   -- Serket Ring (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,4172,@VCOMMON);  -- Reraiser (Very Common, 24%)
+INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,4173,@COMMON);   -- Hi-Reraiser (Common, 15%)
+INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,901,@COMMON);    -- Venomous Claw (Common, 15%)
+INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,4174,@VCOMMON);  -- Vile Elixir (Very Common, 24%)
+INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,12348,@COMMON);  -- Serket Shield (Common, 15%)
+INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,4175,@COMMON);   -- Vile Elixir +1 (Common, 15%)
+INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,16767,@VCOMMON); -- Triple Dagger (Very Common, 24%)
+INSERT INTO `mob_droplist` VALUES (2203,0,0,1000,13552,@COMMON);  -- Serket Ring (Common, 15%)
```

Comment:
I think looking at both the wiki and ffxidb together there seems to be some large differences in the drop tables. I am more inclined to go with the wiki fewere data points (900) but the drop rates seem to be more in-line with what era felt like. Serket ring being a common drop seems extremely unlikely, as well as a low drop rate of vclaw which was extremely common back in the day.
https://ffxiclopedia.fandom.com/wiki/Serket?so=search
http://ffxidb.com/zones/200/serket